src/containers/PaymentInfo/PaymentInfo.container.js

```diff
@@ -37,7 +37,7 @@ export const mapDispatchToProps = dispatch => {
     showInnerPopup: payload => {
       dispatch(showInnerPopup(payload));
     },
-    hideInnerPopup: () => {
+    hidePaymentInfoPopup: () => {
       dispatch(hideInnerPopup());
```

Comment:
as we replaced it with hidePaymentInfoPopup we could use here the new popup manager and new function to hide update payment details popup